en/advanced_config/sensor_thermal_calibration.md

```diff
@@ -12,94 +14,104 @@ The inertial rate gyro and accelerometer sensor offsets are calculated using a 3
 
 Examples:
 
-TC\_G0\_X3\_0 is the ^3 coefficient for the first gyro x-axis
-
-TC\_A1\_TREF is the reference temperature for the second accelerometer
+* [TC_G0_X3_0](../advanced_config/parameter_reference.md#TC_G0_X3_0) is the `^3` coefficient for the first gyro x-axis.
+* [TC_A1_TREF](../advanced_config/parameter_reference.md#TC_A1_TREF) is the reference temperature for the second accelerometer.
 
-## Calibration Parameter Useage
+### Calibration Parameter Usage
 
-The correction for thermal offsets using the calibration parameters is performed in the sensors module.  The reference temperature is subtracted from the measured temperature to obtain a delta temperature where:
-
-delta = measured\_temperature - refernece\_temperature.
+The correction for thermal offsets (using the calibration parameters) is performed in the [sensors module](https://dev.px4.io/en/middleware/modules_system.html#sensors). The reference temperature is subtracted from the measured temperature to obtain a delta temperature where:
+```
+delta = measured_temperature - reference_temperature
+```
 
 The delta temperature is then used to calculate a offset, where:
-
-offset = X0 + X1\*delta + X2\*delta\*\*2 + ... + Xn\*delta\*\*n
+```
+offset = X0 + X1*delta + X2*delta**2 + ... + Xn*delta**n
+```
 
 The offset and temperature scale factor are then used to correct the sensor measurement where:
+```
+corrected_measurement = (raw_measurement - offset) * scale_factor
+```
 
-corrected\_measurement = \(raw\_measurement - offset\) \* scale\_factor
+If the temperature is above the test range set by the `*_TMIN` and `*_TMAX` parameters, then the measured temperature will be clipped to remain within the limits.
 
-If the temperature is above the test range set by the \*\_TMIN and \*\_TMAX parameters, then the measured temperature will be clipped to remain within the limits.
+Correction of the accelerometer, barometers or rate gyroscope data is enabled by setting [TC_A_ENABLE](../advanced_config/parameter_reference.md#TC_A_ENABLE), [TC_B_ENABLE](../advanced_config/parameter_reference.md#TC_B_ENABLE) or [TC_G_ENABLE](../advanced_config/parameter_reference.md#TC_G_ENABLE) parameters to 1 respectively.
 
-Correction os the accelerometer, barometers or rate gyroscope data is enabled by setting TC\_A\_ENABLE\_, \_TC\_B\_ENABLE or TC\_G\_ENABLE parameters to 1 respectively.
+### Compatibility with legacy `CAL_*` parameters and commander controlled calibration
 
-## Compatibility with legacy CAL\_\* parameters and commander controlled calibration
+The legacy temperature-agnostic PX4 rate gyro and accelerometer sensor calibration is performed by the commander module and involves adjusting offset, and in the case of accelerometer calibration, scale factor calibration parameters. The offset and scale factor parameters are applied within the driver for each sensor. These parameters are found in the [CAL parameter group](../advanced_config/parameter_reference.md#sensor-calibration).
```

Comment:
I think what you're saying is that the sensor module applies both legacy and thermal compensation, so if you want to use thermal compensation you should set the "legacy parameters" to zero all the time (both during calibration and compensation). Presumably if you don't want to use thermal compensation you'd set the thermal parameters to zero. Is that correct?

If that is correct, could we use a title like "Disable Legacy CAL_* Parameters" ?